sci-libs/gdal/gdal1-1.11.5.recipe

```diff
@@ -117,13 +121,38 @@ BUILD_PREREQUIRES="
 	"
 
 defineDebugInfoPackage gdal1$secondaryArchSuffix \
-	$libDir/libgdal.so.$libVersion
+	"$commandBinDir"/gdal_contour \
+	"$commandBinDir"/gdal_grid \
+	"$commandBinDir"/gdal_rasterize \
+	"$commandBinDir"/gdal_translate \
+	"$commandBinDir"/gdaladdo \
+	"$commandBinDir"/gdalbuildvrt \
+	"$commandBinDir"/gdaldem \
+	"$commandBinDir"/gdalenhance \
+	"$commandBinDir"/gdalinfo \
+	"$commandBinDir"/gdallocationinfo \
+	"$commandBinDir"/gdalmanage \
+	"$commandBinDir"/gdalserver \
+	"$commandBinDir"/gdalsrsinfo \
+	"$commandBinDir"/gdaltindex \
+	"$commandBinDir"/gdaltransform \
+	"$commandBinDir"/gdalwarp \
+	"$commandBinDir"/nearblack \
+	"$commandBinDir"/ogr2ogr \
+	"$commandBinDir"/ogrinfo \
+	"$commandBinDir"/ogrlineref \
+	"$commandBinDir"/ogrtindex \
+	"$commandBinDir"/testepsg \
+	"$libDir"/libgdal.so.$libVersion
 
 BUILD()
 {
 	expatDir=$portPackageLinksDir/lib~libexpat
 	autoreconf -fi
-	runConfigure ./configure --with-expat=$expatDir
+	runConfigure --omit-dirs binDir ./configure \
+		--bindir $commandBinDir \
+		--with-expat=$expatDir \
+		--with-static-proj4=yes
```

Comment:
With a static proj4, you probably don't need "lib:libproj" in REQUIRES.

Reply:
removed that :) so rebuilding also against newer libproj package, so also here, just a sec :)